string graph_name
bool name_match_required
string[] targets
string umrf_graph_json

temoto_action_engine/UmrfGraphDiff[] umrf_graph_diffs
